arch/xtensa/src/common/xtensa_exit.c

```diff
@@ -68,7 +68,7 @@ void up_exit(int status)
 
   /* Scheduler parameters will update inside syscall */
```

Comment:
Does this comment still valid after this modification? I don't think so!

Reply:
Updated